source/ui/dataman/AMScanThumbnailGridView.cpp

```diff
@@ -4,415 +4,443 @@
 #include "ui/dataman/AMScanThumbnailGridInputManager.h"
 
 AMScanThumbnailGridView::AMScanThumbnailGridView(QWidget *parent) :
-	QAbstractItemView(parent)
```

Comment:
Either your spacing is messed up now, or mine was messed up when I first wrote this code. My guess is the latter. You don't need to do anything with this code, but do remind me to check your settings.